app/models/student_application.rb

```diff
@@ -35,4 +35,7 @@ class StudentApplication < ActiveRecord::Base
   validates :why_join, presence: true
   validates :phone, presence: true
   validates :year, presence: true
+  validates :name, presence: true
+
+  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
```

Comment:
Prefer the new style validations `validates :column, format: value` over `validates_format_of`.